root/graf/tasimage_frompad.C

```diff
@@ -0,0 +1,30 @@
+
+int tasimage_frompad()
+{
+   auto GetVMem = []() {
+      ProcInfo_t info;
+      gSystem->GetProcInfo(&info);
+      return float(info.fMemVirtual);
+   };
+
+   gROOT->SetBatch();
+
+   auto img = TImage::Create();
+   const auto initVmem = GetVMem();
+   for (auto i : ROOT::TSeqI(5000)) {
+      TCanvas pad;
+      img->FromPad(&pad);
+   }
+   usleep(30); // give time to the counters to be updated
```

Comment:
```suggestion
   gSystem->Sleep(1); // give time to the counters to be updated
```
if 1ms is not too long 😉